modules/boards/app/controllers/boards/boards_controller.rb

```diff
@@ -1,6 +1,9 @@
 module ::Boards
   class BoardsController < BaseController
+    include ::BoardsHelper
```

Comment:
You don't need to include the helper, you could just use `helpers.<method>`?

Reply:
You're correct.

As I just wrote in the commit message to remove it:

> I was mislead into this being needed during my testing as `BoardsHelper`
> didn't seem to be available within the view but it seemed to be because
> there was no `helpers` directory initially so it wasn't going to be autoloaded
> until I stopped spring and restarted the rails server.